airbyte-integrations/connectors/source-segment/metadata.yaml

```diff
@@ -0,0 +1,35 @@
+metadataSpecVersion: "1.0"
+data:
+  allowedHosts:
+    hosts:
+      - "https:"
```

Comment:
Welp, looks like I have a bug there in my code — second time I see `https://` here instead of the actual domain.

Reply:
@zckymc can you get the icon.svg and include here?